src/EnergyPlus/InputProcessor.cc

```diff
@@ -5315,9 +5315,8 @@ namespace InputProcessor {
 		int Loop;
 		int Loop1;
 
-		OutputVariablesForSimulation.allocate( 10000 );
+		OutputVariablesForSimulation.reserve( 1024 );
```

Comment:
No, just didn't want 10000 and wanted a base 2 number. Since it is just a reserve, it doesn't matter too much.